jscript/script.js

```diff
@@ -0,0 +1,238 @@
+const courses = [
+  {
+    name: 'PROGRAMMING WITH SCRATCH',
+    classes: [
+      {
+        name: 'Module1',
+      },
+      {
+        name: 'Module2',
+      },
+      {
+        name: ' Project',
+      },
+    ],
+    img: 'pics/scratch.png',
+
+  },
+
+  {
+    name: 'WEB DEVELOPMENT ',
+    classes: [
+      {
+        name: 'Module1',
+      },
+      {
+        name: 'Module2',
+      },
+      {
+        name: ' Project',
+      },
+    ],
+    img: 'pics/hcss.png',
+  },
+
+  {
+    name: 'PROGRAMMING WITH PYTHON',
+    classes: [
+      {
+        name: 'Module1',
+      },
+      {
+        name: 'Module2',
+      },
+      {
+        name: ' Project',
+      },
+    ],
+    img: 'pics/pyth1.png',
+  },
+
+  {
+    name: 'CIRCUITS DESIGN',
+    classes: [
+      {
+        name: 'Module1',
+      },
+      {
+        name: 'Module2',
+      },
+      {
+        name: ' Project',
+      },
+    ],
+    img: 'pics/elect3.png',
+  },
+];
+
+const orgpart1 = document.querySelector('#home-container');
+const title = document.createElement('h2');
+orgpart1.appendChild(title);
+title.textContent = 'Courses offered';
+const lineh = document.createElement('hr');
+title.appendChild(lineh);
+
+const para = document.createElement('p');
+para.classList.add('p-present');
+orgpart1.appendChild(para);
+para.textContent = 'Our digital making projects have step-by-step instructions to teach you how to think computatioanlly, then create games, animations and much more';
+
+const gridimg2 = document.createElement('div');
+gridimg2.classList.add('containerhome');
+orgpart1.appendChild(gridimg2);
+
+courses.forEach((part) => {
+  const divpart1 = document.createElement('div');
+  divpart1.classList.add('subdivpart1');
+  gridimg2.appendChild(divpart1);
+  const moduleList = document.createElement('ul');
+  moduleList.classList.add('mod-items');
+
+  const part1 = document.createElement('div');
+  part1.classList.add('icons');
+  divpart1.appendChild(part1);
+
+  const imgdiv = document.createElement('img');
+  imgdiv.src = part.img;
+  imgdiv.classList.add('dynamicimg1');
+  part1.appendChild(imgdiv);
+
+  const divh = document.createElement('h2');
+  part1.appendChild(divh);
+  divh.textContent = part.name;
+
+  part1.appendChild(moduleList);
+  part.classes.forEach((type) => {
+    const itemL = document.createElement('li');
+    moduleList.appendChild(itemL);
+    itemL.textContent = type.name;
+  });
+});
+
+const btnclick = document.createElement('button');
+btnclick.classList.add('join-btn');
+orgpart1.appendChild(btnclick);
+btnclick.textContent = 'Join  DigiKidTech';
+
```

Comment:
Guideline
![image](https://github.com/hydaline-code/Capstone_project_1/assets/78251927/d1d50269-5b8c-4ab9-a077-3b54f8d4b3c2)

Your design
![image](https://github.com/hydaline-code/Capstone_project_1/assets/78251927/2f94a138-aa04-4574-b9b9-e04a42d250ff)

I recommend that we change the text color of the text `Join DigiKidTech` to `white` for a good contrast ratio between the text and its background to ensure that the text is easy to read for all users, including those with visual impairments.